matchms/importing/load_from_usi.py

```diff
@@ -0,0 +1,48 @@
+import json
+import numpy as np
+import requests
+from matchms import Spectrum
```

Comment:
This imports results in Spectrum being a module instead of class.
```suggestion
from ..Spectrum import Spectrum
```